infrastructure/make/go/include.mk

```
@@ -17,32 +17,32 @@ MAKEFLAGS += --no-builtin-rules
 .PHONY: deps
 deps:
 ifeq ($(SKIP_DEPS),0)
```

```suggestion
ifeq ($(SKIP_BUILDER),0)
```
please rename this, the meaning has changed